### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -1,64 +1,36 @@
...
+- Lars Holm Nielsen
+- Leonardo Rossi
+- Liam Kirsh
+- Nicolas Harraudeau
+- Odd
+- Orestis Melkonian
+- Rémi Ducceschi
+- Sami Hiltunen
+- Sebastian Witowski
+- Tibor Simko
```
Comment (lnielsen) [inline AUTHORS.rst]: Contributors issues

Comment (switowski) [reply AUTHORS.rst]: You mean Odd? Or is there something else missing?

Comment (lnielsen) [reply AUTHORS.rst]: It's too long....start commit should be f55f1a909c37cb70d9570ca1d9830269464ad4cd

Comment (switowski) [reply AUTHORS.rst]: Hmm, from the list you gave me (https://github.com/inveniosoftware/license-changer/blob/master/external_contributors_stats.py#L7), starting commit is f97176828348baeb32bf9556001112ba49bcf4b2

Comment (switowski) [reply AUTHORS.rst]: Ok, fixed, it was just Adrian who had to be removed.

**Pull request merged**